go.mod

```diff
@@ -155,7 +155,7 @@ require (
 	gopkg.in/square/go-jose.v2 v2.3.1
 	gopkg.in/stack.v1 v1.7.0 // indirect
 	gopkg.in/yaml.v2 v2.2.2
-	istio.io/api v0.0.0-20190813012407-93fc23ae4a32
+	istio.io/api v0.0.0-20190820204432-483f2547d882
```

Comment:
This doesn't match the version in `istio/proxy` for the SHA you chose in `istio.deps`:
https://github.com/istio/proxy/blob/3245d3582d7e4937150b2c56120a556420721ac3/istio.deps
It should be `36b42252042c0e8b229a0d66059af184c3105f9d `.

Reply:
Discussed with @mandarjog. The API SHA in `istio/proxy:release-1.3` points to a commit of `istio/api` that is >1 month old. This needs to be updated first to use the same version as in `istio/istio:release-1.3`.

Reply:
https://github.com/istio/proxy/pull/2384